.github/workflows/cd-publish-docs-branch.yml

```diff
@@ -0,0 +1,78 @@
+---
+# yamllint disable line-length
+name: "Publish docs to GitHub Pages"
+
+on: # yamllint disable-line rule:truthy
+  workflow_dispatch:
```

Comment:
Did we want this on workflow_dispatch, or when the docs folder is merged to main after the automated PR?